### ISSUE TITLE
Add a reminder to acknowledge the ODA paper at ODA frontend

### ISSUE THREAD

**Issue by andriineronov** [oda-hub/frontend-chart#1]
Add a reminder to acknowledge the ODA paper at ODA frontend. As discussed, perhaps somewhere just below the grey field with session ID

Add
Subtasks

Add
Related issues

History

#1 Updated by Mohamed Meharga about 1 year ago

Status changed from New to In Progress
  
The same apply to this task as for the task #19745, Create a new "Science/Publicaitons" tab at CDCI webpage

It can be implemented quickly.
It should be done once the link to the paper is ready.

#2 Updated by Mohamed Meharga 10 months ago

Status changed from In Progress to On hold
Assignee changed from Mohamed Meharga to Andrii Neronov
  
Waiting for the paper link !

#3 Updated by Volodymyr Savchenko 10 months ago

Assignee changed from Andrii Neronov to Mohamed Meharga
  
here is the link:

https://arxiv.org/abs/2002.12895

it's not A&A link yet, but we all know that very well, so I strongly suspect it is meant to put this link, without waiting for referring (which might take monthes).

#4 Updated by Mohamed Meharga 10 months ago

Status changed from On hold to In Progress
#5 Updated by Mohamed Meharga 4 months ago

% Done changed from 0 to 90
#6 Updated by Andrii Neronov 2 months ago

Target version changed from ODA 1.2 to ODA 1.3
